application/routes/audio.py

```
@@ -35,22 +35,27 @@
 from .. import api
 from ..codes import ConnectionType
 from ..features.Audio import AUDIO_PORT
-
+import logging
+logger = logging.getLogger(__name__)
 
 @api.route('/audio', methods=['POST'])
 def start_audio():
+    #TODO: Request recieved with body
+    logger.debug("Audio: Received request to start audio with data: {}".format(request.json))
```

Ruff

Use f-string instead of `format` call

Convert to f-string

(UP032)